seahorn/jobs2/priority_queue_capacity2/sea.yaml

```diff
@@ -0,0 +1,2 @@
+verify_options:
+  horn-bv2-tracking-mem: true
```

Comment:
Github isn't showing this, but I have a newline here already. Ditto for the other sea.yaml files.